Sources/SwiftProtobuf/ProtobufUtility.swift

```diff
@@ -17,7 +17,7 @@
 
 import Swift
 
-public func ==(lhs: [[UInt8]], rhs: [[UInt8]]) -> Bool {
+func ==(lhs: [[UInt8]], rhs: [[UInt8]]) -> Bool {
```

Comment:
FYI:  It's possible these `[[UInt8]]` helpers can just be dropped.  They're mostly leftovers from when `bytes` was being rendered as `[UInt8]` but now we're mapping `bytes` to `Data`, so these may no longer be used at all.